README.md

```diff
@@ -71,7 +79,7 @@ Usage: cardano-node --topology FILEPATH --database-path FILEPATH
 
 `--port` - Specify which port to assign to the node.
 
-`--config` - Specify the filepath to the config `.yaml` file. This file is responsible for all the other node's required settings. See examples in `configuration` (e.g. `log-config-0.yaml`).
+`--config` - Specify the filepath to the config `.yaml` file. This file is responsible for all the other node's required settings. See examples in `configuration` (e.g. `config-0.yaml`).
```

Comment:
Maybe add a link to `configuration/defaults/simpleview/config-0.yaml` so user can click and check it right a way.